docker-for-windows/index.md

```diff
@@ -443,64 +441,66 @@ to change context so that `kubectl` is pointing to `docker-for-desktop`:
 > kubectl config use-context docker-for-desktop
 ```
 
-You can also change it through the Docker Desktop for Windows menu:
+You can also change it through the Docker Desktop menu:
 
-![Change Kubernetes Context](images/docker-menu-context-switch.png){:width="600px"}
+![Change Kubernetes Context](images/docker-menu-context-switch.png){:width="450px"}
 
 If you installed `kubectl` by another method, and
 experience conflicts, remove it.
 
 - To enable Kubernetes support and install a standalone instance of Kubernetes
   running as a Docker container, select **Enable Kubernetes** and click the
-  **Apply and restart** button.
-
-  An Internet connection is required. Images required to run the Kubernetes
-  server are downloaded and instantiated as containers, and the
-  > Program Files\Docker\Docker\Resources\bin\kubectl.exe` command is installed.
+  **Apply** button. This instantiates images required to run the Kubernetes server as containers, and installs the `kubectl.exe` command in the path.
 
 - By default, Kubernetes containers are hidden from commands like `docker
   service ls`, because managing them manually is not supported. To make them
-  visible, select **Show system containers (advanced)** and click **Apply and restart**.
+  visible, select **Show system containers (advanced)** and click **Apply**.
   Most users do not need this option.
 
 - To disable Kubernetes support at any time, deselect **Enable Kubernetes**.
   The Kubernetes containers are stopped and removed, and the
   `/usr/local/bin/kubectl` command is removed.
 
-  For more about using the Kubernetes integration with Docker Desktop for Windows,
-  see [Deploy on Kubernetes](kubernetes.md).
+  For more about using the Kubernetes integration with Docker Desktop, see [Deploy on Kubernetes](kubernetes.md).
```

Comment:
Suggest "For more information on using the Kubernetes..."

Reply:
Good catch.

Reply:
Fixed in commit 59aa7e5.